dev/fattest.simplicity/src/componenttest/annotation/AllowedFFDC.java

```diff
@@ -21,11 +21,13 @@
 public @interface AllowedFFDC {
 
     public static final String ALL_FFDC = "ALL_FFDC";
+    public static final String ALL_REPEAT_ACTIONS = "ALLREPEATACTIONS";
 
     /**
      * A string array fully-qualified FFDC exception class names.
      * If no value is specified, any FFDCs are allowed.
      */
     String[] value() default { ALL_FFDC };
 
+    String repeatAction() default ALL_REPEAT_ACTIONS;
```

Comment:
Please add some javadoc that describes how this attr works here and in `ExpectedFFDC`

Reply:
Also, should this be an array of String instead of a single `String`?